config/federatedtypes/deployment.yaml

```diff
@@ -17,6 +17,6 @@ spec:
     kind: FederatedDeploymentPlacement
   override:
     kind: FederatedDeploymentOverride
-  overridePath:
-    - spec
-    - replicas
+  overridePaths:
+    - path: spec.replicas
+    - path: spec.revisionHistoryLimit
```

Comment:
I appreciate your desire to test multiple overrides with the crud tester, but this type config will be deployed (it's not test-only).  Once #340 lands it will be trivial to generate a type config + crds to enable this testing, can the testing wait until then?

Reply:
Sounds ok to me, Updated!
Meanwhile, I am working my way to submit tests for job overrides and service overrides (multiple fields) but I can wait to use `kubefed federated` (which I did look up after you mentioned it.. :) ). As far as I understood some additional test yamls would still be needed though (stuff currently in test/common/fixtures). The ones already there might be good for default crud test; What would be your suggestion on updating them vs using newer ones for specific tests?

Reply:
It should be possible to use the crd test in #340 to test multiple overrides against a generated crd rather than requiring test fixture.  I should have something for overrides up later today now that this is ready to merge.